docs/conversion.md

```diff
@@ -0,0 +1,64 @@
+This document outlines all the conversion details regarding `docker-compose.yaml` values to Kubernetes / OpenShift artifacts.
+
+| Value             | Version   | Kompose     | K8s / OpenShift                                                                                                                                                                                                                                                          | Notes                                                                                                                       |
+|-------------------|-----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+|                   |           |             |                                                                                                                                                                                                                                                                          |                                                                                                                             |
+| Service           |           |             |                                                                                                                                                                                                                                                                          |                                                                                                                             |
+| build             |           | YES         | [BuildConfig](https://docs.openshift.com/enterprise/3.1/dev_guide/builds.html#defining-a-buildconfig)                                                                                                                                                                    | Converts, but local builds are not yet supported. See issue [97](https://github.com/kubernetes-incubator/kompose/issues/97) |
+| cap_add, cap_drop |           | YES         | N/A                                                                                                                                                                                                                                                                      |                                                                                                                             |
```

Comment:
I think there is a mistake in the code. We are reading cap_add and cap_drop but we are not using it anywhere.

PR https://github.com/kubernetes-incubator/kompose/pull/468 is adding `cap_add` and `cap_drop` to  unsupported keys.